datacatalog/cloud-client/src/main/java/com/example/datacatalog/QuickStart.java

```diff
@@ -0,0 +1,4 @@
+package com.example.datacatalog;
+
+public class QuickStart {
```

Comment:
Is this file supposed to be here?

Reply:
Sorry, it's a WIP accidentally pushed. Just removed it...